packages/web-components/src/components/rux-container/rux-container.scss

```diff
@@ -0,0 +1,39 @@
+:host {
+    display: block;
+    --container-header-padding: 16px 20px;
```

Comment:
trying something new here and creating private component tokens. These are NOT css custom props and should not be documented. These will eventually be added into our design tokens. This is more or less just a way to document them before we get to that.